### ISSUE TITLE
Rotatable trait doesnt work in version 1.3.0

### ISSUE THREAD

**Issue by LuciaDX** [CustomiesDevs/Customies#78]
Everything was working fine in 1.2.0 under pm4 but now these blocks dont rotate

**Comment by xSuperr** [comment]
+1 for 1.20 PM5

**Comment by CoolCuzzy** [comment]
Any solution found for this problem! Really urgent and it’s a really BIG problem.

**Comment by TwistedAsylumMC** [comment]
Could you try changing this line to use "states" rather than "properties"?
```diff
- ->setTag("properties", new ListTag(array_reverse($blockProperties))); // fix client-side order
+ ->setTag("states", new ListTag(array_reverse($blockProperties))); // fix client-side order
```

https://github.com/CustomiesDevs/Customies/blob/master/src/block/CustomiesBlockFactory.php#L130

**Comment by royaljacques** [comment]
i have looking this in the bedrock.dev :
https://wiki.bedrock.dev/blocks/rotatable-blocks.html#facing-rotation-setup
```json
"permutations": [
      {
        "condition": "query.block_property('wiki:rotation') == 0",
        "components": {
          "minecraft:transformation": { "rotation": [0, 0, 0] }
        }
      },
      {
        "condition": "query.block_property('wiki:rotation') == 1",
        "components": {
          "minecraft:transformation": { "rotation": [90, 0, 0] }
        }
      },
      {
        "condition": "query.block_property('wiki:rotation') == 2",
        "components": {
          "minecraft:transformation": { "rotation": [0, 0, 90] }
        }
      }
```

**Comment by LuciaDX** [comment]
> Could you try changing this line to use "states" rather than "properties"?
> 
> ```diff
> - ->setTag("properties", new ListTag(array_reverse($blockProperties))); // fix client-side order
> + ->setTag("states", new ListTag(array_reverse($blockProperties))); // fix client-side order
> ```
> 
> https://github.com/CustomiesDevs/Customies/blob/master/src/block/CustomiesBlockFactory.php#L130

![image](https://github.com/CustomiesDevs/Customies/assets/82135652/57213f3a-d916-4113-b8c6-09139c7b520d)
This is the result

**Comment by TwistedAsylumMC** [comment]
This should be fixed by https://github.com/CustomiesDevs/Customies/pull/79, if anyone can test this it would be very appreciated

**Comment by LuciaDX** [comment]
> This should be fixed by #79, if anyone can test this it would be very appreciated

Now every block that has trait just went invisible
![image](https://github.com/CustomiesDevs/Customies/assets/82135652/d3ca440a-28d9-495b-b44b-1a765140498c)

**Comment by CoolCuzzy** [comment]
Yeah issue not resolved.
